gradle.properties

```diff
@@ -8,7 +8,7 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 ### Pravega dependencies
-pravegaVersion=0.6.0-50.076afef-SNAPSHOT
+pravegaVersion=0.6.0-2397.5a60e36-SNAPSHOT
```

Comment:
This tag is not publicly available, we need to replace it with the latest artifact published in JFrog repo: https://oss.jfrog.org/artifactory/jfrog-dependencies/io/pravega/